cnd/src/swap_protocols/rfc003/spawn.rs

```diff
@@ -1,62 +1,58 @@
 use crate::{
-    connector::Connector,
+    dependencies::Dependencies,
     swap_protocols::{
         asset::Asset,
         rfc003::{
-            self,
-            alice::{self, State, SwapCommunication},
-            state_machine,
-            state_store::StateStore,
-            CreateLedgerEvents, Ledger,
+            alice::{State, SwapCommunication},
+            state_machine::{self, SwapStates},
+            CreateLedgerEvents, Ledger, Request, Response,
         },
         LedgerConnectors,
     },
 };
-use futures::{sync::mpsc, Stream};
+use futures::sync::mpsc;
 use futures_core::{
     compat::Future01CompatExt,
     future::{FutureExt, TryFutureExt},
 };
 use std::sync::Arc;
 
-pub trait SpawnAlice: Send + Sync + 'static {
+pub trait Spawn: Send + Sync + 'static {
```

Comment:
We've started to talk about an "execution layer" in COMIT. This trait is basically representing that. What do you think about renaming it to `ExecuteSwap` or `SwapExecutor` (analogous to `TaskExecutor`).

Reply:
I may have missed something in that discussion but I am under the impression that the request/response COMIT messages are still grouped under the execution phase (probably need renaming if we do add a separate negotiation protocol.  Happy to rename but may need further thought.

Reply:
No need for comment, discussed offline.